### PR TITLE
add the posibility to install katello_agent

### PR DESCRIPTION
**Pull request by cristifalcas**
The package is in katello repo, so the user will have to make it available somehow

### CODE REVIEW

**Comment by ehelms** [comment]
This looks like it doesn't install katello-agent but rather our pulp-katello plugin which is really just a yum_clone distributor (with a poorly named repository).

**Comment by cristifalcas** [comment]
Ah, yes. It's for installing pulp-katello package.

File: manifests/install.pp [hunk 1]
```diff
@@ -17,6 +17,14 @@
     )
   }
 
+  if $pulp::enable_katello {
+    if $pulp::messaging_transport == 'rabbitmq' {
+      fail('pulp-katello plugin doesn\'t support rabbitmq')
```
Comment (ehelms) [inline manifests/install.pp]: I don't think this is true? Given that the `pulp-katello` plugin is just adding a distributor type to Pulp.

Comment (cristifalcas) [inline manifests/install.pp]: Correct. I thought initially that it is connected to katello somehow via qpid.

**Comment by ehelms** [comment]
Looking good to me - mind squashing the commits and I'll merge?

**Comment by ehelms** [comment]
Thanks @cristifalcas !

**Comment by ehelms** [comment]
@cristifalcas If you need an official release with your new provides and updates just let me know

**Pull request merged**